scripts/helper_numpy.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# coding: utf8
+'''Compute a Numpy friendly data-structures from VGGish files
+Example
+-------
+$ cd {repo_root}
+$ ./scripts/helper_numpy.py --csv_file /path/to/sparse-labels.csv
+--vggish_path /path/to/vggish/ --output_file /path/to/output.npz
+
+Any valid output file '*.npz' has to be speficied by the user
+'''
+
+import argparse
+import numpy as np
+import os
+import pandas as pd
+import sys
+import json
+
+
+def main(csvfile, vggishpath, outfile):
+
+    success = []
+    df = pd.read_csv(csvfile)
+    instruments = np.unique(df['instrument'])
+    sample_key = np.unique(df['sample_key'])
+
+    print('Extracting the vggish features...')
+    X = np.empty([20000, 10, 128], dtype=int)
+    count = 0
+    for sk in sample_key:
+        sk_prefix = sk[:3]
+        full_name = vggishpath + sk_prefix + '/' + sk + '.json'
```

Comment:
channeling @bmcfee it's generally preferred to use `os.path.join` and `os.path.sep` to build filepaths, since some of this could change depending on platform being used

Reply:
I use `os.path.join` now, but not `os.path.sep`. If you think that is important, feel free to suggest something.